openlibrary/templates/books/custom_carousel.html

```diff
@@ -7,6 +7,21 @@
   $if slick_font not in ctx.links:
     $ ctx.links.append(slick_font)
 
+$def carousel_i18n_input():
+  $ show_carousel_i18n_strings=  {
+  $ "open": _("Read"),
+  $ "borrow_available": _("Borrow"),
+  $ "borrow_unavailable": _("Join Waitlist"),
+  $ "error": _("Not in Library"),
+  $ "loading": _("Loading...")
+  $ }
+
+  <input type="hidden" name="carousel-i18n-strings" value="$json_encode(show_carousel_i18n_strings)">
+
+
+  $if render_once('books/custom_carousel.i18n-strings')::
+  $:carousel_i18n_input()
```

Comment:
```suggestion
  $if render_once('books/custom_carousel.i18n-strings'):
    $:carousel_i18n_input()
```
Second colon causes an error.
Be sure to indent nested statements.